### DISCUSSION TITLE
Marked Off-Topic

### DISCUSSION THREAD

**Discussion by Sir-Alex-L** [TeamNewPipe/NewPipe#10286] [Questions] [answered]
Confused as why my comment was marked off-topic at #10284 

It's RC with merge  #1084.
Do you want me to comment in the merge itself? Or the bug report #10174?

**Comment by SameenAhnaf** [comment, marked as answer]
Release issues only care about NEW regressions reproducible in debug APKs. Your comment included bugs which could be reproduced in previous versions and already reported in #10292 and https://github.com/TeamNewPipe/NewPipe/issues/8318

**Reply by Sir-Alex-L** [reply]
Thanks, I thought it was testing the fix. I deleted the comment entirely.
